OpenRA.Mods.Common/UtilityCommands/UpgradeMapCommand.cs

```diff
@@ -37,9 +37,16 @@ static void ProcessYaml(Map map, MiniYaml yaml, int engineDate, UpgradeAction pr
 				var files = FieldLoader.GetValue<string[]>("value", yaml.Value);
 			    foreach (var filename in files)
 			    {
```

Comment:
While you're here, could you also please remove the spaces in this line and the one above?